### PR TITLE
Translate all Dutch HTML texts

### PR DESCRIPTION
**Pull request by se-bastiaan**

### CODE REVIEW

File: orders/templates/orders/index.html [hunk 1]
```diff
@@ -16,22 +16,22 @@
     <div class="container-fluid">
         <div class="row">
             <div class="col-sm-4 sidebar">
-                <h2>Bestelling</h2>
+                <h2>Order</h2>
                 <form role="form" method="post">
                     {% csrf_token %}
                     {% bootstrap_messages %}
                     <p>Hoe wil je betalen?</p>
```
Comment (pingiun) [inline orders/templates/orders/index.html]:
```suggestion
                     <p>How do you want to pay?</p>
```

Comment (pingiun) [reply orders/templates/orders/index.html]: I mean "How to pay here for?"

**Review by pingiun — DISMISSED** [review]

File: orders/templates/orders/overview.html [hunk 2]
```diff
@@ -18,45 +18,44 @@
...
                                     <i class="fa fa-slack"></i>
                                     Slack
                             </button>
                         {% endif %}
-                        Zie ook het
+                        If you want you can make a
                         <a class="btn btn-default btn-xs" role="button" href="{% url 'summary' %}">
                             {% bootstrap_icon 'print' %}
-                            printvriendelijke overzicht
+                            printable overivew
```
Comment (joren485) [inline orders/templates/orders/overview.html]:
```suggestion
                            printable overview
```

File: orders/templates/orders/overview.html [hunk 3]
```diff
@@ -18,45 +18,44 @@
...
+                        order list then click
                         <button type="submit" name="process" value="all"
                                 class="btn btn-success btn-xs"
-                                onclick="return confirm('Weet je zeker dat je de bestellingen wilt verwerken?')">
-                                alles verwerken {% bootstrap_icon 'tag' %}
+                                onclick="return confirm('Are you sure you want to process all orders?')">
+                                process all orders {% bootstrap_icon 'tag' %}
                         </button>
-                        om de lijst leeg te maken en een bonnetje te genereren.
+                        to empty the list and generate a receipt.
```
Comment (joren485) [inline orders/templates/orders/overview.html]:
```suggestion
                        to clear the list and generate a receipt.
```

**Review by joren485 — APPROVED** [review]

**Pull request merged**